docs/source/contributing.rst

```diff
@@ -0,0 +1,187 @@
+Contributing to Kordac
+#######################################
+
+The git repository for Kordac can be found here_.
+
+.. _here: https://github.com/uccser/kordac
+
+
+Issue Reporting and Bug Fixes
+=======================================
+If you come across a bug in Kordac, please `report it`_ on the repo.
+
+<Bug fix? Add tests.>
+
+.. _report it: https://github.com/uccser/kordac/issues
+
+The Code Base
+=======================================
+If you would like to contribute to Kordac, fork the repository.
+
+Overview
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Before reading this section, make sure you have read `how to use Kordac`_ (or even better, have already used Kordac!).
+
+.. _how to use Kordac: http://kordac.readthedocs.io/en/develop/usage.html
+
+
+There are several parts and terms of Kordac to become familiar with:
+
+- **Tag**
+    This refers to the custom markdown syntax that Kordac processes.
+    
+    For example:
+    
+    .. code-block:: none
+
+      {comment this will be removed by the converter}
+
+      {image file-path='img/totally-real-image.png' alt='process me'}
+    
+    are examples of the ``comment`` and ``image`` tags in Kordac.  
+
+- ``Kordac()``
+	The convertor object itself. This is what a user will use to create a Kordac converter, and what is used to define a custom processor list, custom html templates and custom Markdown Extensions to use.
+
+- ``KordacResult()``
+    The object returned by ``Kordac()`` containing:
+    	- Converted html string
+    	- Title
+    	- Required files (images, interactives, scratch images, page scripts)
+    	- Heading tree
+    	- Required glossary terms
+
+- ``KordacExtension()``
+    Inherits the ``Extension`` class from Markdown.
+    This class is the main class of the project. It loads all the processor information, loads the template files and clears and populates the attributes to be returned by the ``KordacResult`` object.
+
+- ``Processors/``
+  	There is a different processor for each tag. A processor uses it's corresponding regex loaded from ``processor-info.json`` to find matches in the text, and uses the given arguments in the matched tag to populate and output it's html template.
+
+- ``html-templates/``
+  	The html templates (using Jinja2 template engine) to be populated by processors.
+
+- ``processor-info.json``
+	Every processor is listed in this file, and will at least contain a regex pattern to match it's corresponding tag.
+	Most will also define required and optional parameters, these correspond to arguments in the tag's template.
+
+
+
+Creating a New Processor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To create a new processor, a good place to start is the `Extension API`_ page of the Python Markdown docs, or even the `source code`_ itself.
```

Comment:
Suggestion: 

> or even the `source code`_ itself.

to

> and you can also read the `source code`_ itself.

The documentation should be required reading rather than optional.